tools/jvm/src/main/scala/org/scalajs/core/tools/sourcemap/SourceMapper.scala

```diff
@@ -1,88 +0,0 @@
-/*                     __                                               *\
-**     ________ ___   / /  ___      __ ____  Scala.js sbt plugin        **
-**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
-**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
-** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
-**                          |/____/                                     **
-\*                                                                      */
-
-
-package org.scalajs.core.tools.sourcemap
-
-import org.scalajs.core.tools.classpath._
-
-import com.google.debugging.sourcemap._
-
-class SourceMapper(classpath: CompleteClasspath) {
```

Comment:
This class disappears, but I don't see any *usage* of this class in the entire diff. Was it dead code already?